tests/unit/drm_plugin_test.sh

```diff
@@ -225,16 +233,99 @@ function test_gui_control_remote()
   compare_command_sequence '' "$LINENO" 'expected_cmd_seq' "$output"
 }
 
+function test_gui_control_after_reboot_local()
+{
+  local gui_on_after_reboot_cmd='systemctl set-default graphical.target'
+  local gui_off_after_reboot_cmd='systemctl set-default multi-user.target'
+  local bind_cmd='for i in /sys/class/vtconsole/*/bind; do printf "%s\n" 1 > $i; done; sleep 0.5'
+  local unbind_cmd='for i in /sys/class/vtconsole/*/bind; do printf "%s\n" 0 > $i; done; sleep 0.5'
```

Comment:
Maybe I'm wrong, but I think these commands are used in other parts of the test. Instead of duplicating them, we can try to reuse them.

Reply:
Done.